lib/gradelib.php

```diff
@@ -925,7 +925,7 @@ function grade_get_categories_menu($courseid, $includenew=false) {
     }
     foreach ($categories as $key=>$category) {
         if ($category->is_course_category()) {
-            $result[$category->id] = get_string('uncategorised', 'grades');
+            $result[$category->id] = get_string('selectcategory', 'local_gugcat');
```

Comment:
I think this is part of the standard Moodle library. Please do not modify this.

Reply:
changes in gradelib is reverted in the latest commit [5e3d7c1](https://github.com/uofg-andf/andfmoodle/pull/542/commits/5e3d7c1d0e23af1f23e6a6d286e0d0d78c7f6d79)